engines/engine.cpp

```diff
@@ -421,7 +421,7 @@ void initGraphics(int width, int height) {
 void initGraphics3d(int width, int height) {
 	g_system->beginGFXTransaction();
 		g_system->setGraphicsMode(0, OSystem::kGfxModeRender3d);
-		g_system->initSize(width, height);
+		g_system->initSize(width, height, Graphics::PixelFormat::createFormatCLUT8());
```

Comment:
It's a bit strange to request CLUT8 for 3D games. Since the 3D graphics manager ignore that parameters, maybe it would make more sense to pass a null pixel format with `Graphics::PixelFormat()`.